### PR TITLE
[WIP] - Portuguese Translation

### PR DESCRIPTION
**Pull request by gabsprates**
I started this translation in Brazilian Portuguese.

- [x] Basic Matchers
- [x] Meta character
  - [x] Full stop
  - [x] Character set
    - [x] Negated character set
  - [x] Repetitions
    - [x] The Star
    - [x] The Plus
    - [x] The Question Mark
  - [x] Braces
  - [x] Character Group
  - [x] Alternation
  - [x] Escaping special character
  - [x] Anchors
    - [x] Caret
    - [x] Dollar
- [x] Shorthand Character Sets
- [x] Lookaround
  - [x] Positive Lookahead
  - [x] Negative Lookahead
  - [x] Positive Lookbehind
  - [x] Negative Lookbehind
- [x] Flags
  - [x] Case Insensitive
  - [x] Global search
  - [x] Multiline

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
 * [English](README.md)
 * [中文版](README-cn.md)
```
Comment (eamanu) [inline README.md]: Add spanish version

File: README-pt_BR.md [hunk 2]
```diff
@@ -0,0 +1,475 @@
...
+
+<pre>
+"/.at(.)?$/gm" => The <a href="#learn-regex"><strong>fat</strong></a>
+                  cat <a href="#learn-regex"><strong>sat</strong></a>
+                  on the <a href="#learn-regex"><strong>mat.</strong></a>
+</pre>
+
+[Teste a RegExp](https://regex101.com/r/E88WE2/1)
+
+## Bônus
```
Comment (eamanu) [inline README-pt_BR.md]: Take care that the Bonus section was removed

Comment (gabsprates) [inline README-pt_BR.md]: Thanks! :smile:

**Comment by eamanu** [comment]
Sorry Not ready yet

File: README-pt_BR.md [hunk 3]
```diff
@@ -0,0 +1,434 @@
...
+
+> Expressão Regular é um grupo de caracteres ou símbolos utilizado para encontrar um padrão específico a partir de um texto.
+
+Uma expressão regular é um padrão que é comparado com uma cadeia de caracteres da esquerda para a direita. A expressão "Expressão regular" é longa e difícil de falar, você geralmente vai encontrar o termo abreviado como "regex" ou "regexp". Expressões regulares são usadas para substituir um texto dentro de uma string, validar formulários, extrair uma parte de uma string baseada em um padrão encontrado e muito mais.
+
+Imagine que você está escrevendo uma aplicação e quer colocar regras para quando um usuário escolher seu username. Nós queremos permitir que o username contenha letras, números, underlines e hífens. Nós também queremos limitar o número de caracteres para não ficar muito feio. Então usamos a seguinte expressão regular para validar o username:
+
+<br/><br/>
+<p align="center">
+<img src="https://i.imgur.com/ekFpQUg.png" alt="Regular expression">
```
Comment (eamanu) [inline README-pt_BR.md]: Could you put this image on Portuguese?

Comment (gabsprates) [inline README-pt_BR.md]: Of course! But my plan is to do it in the end. :sweat_smile:

**Comment by gabsprates** [comment]
Ok @eamanu, I'm ready now. But, how do I resolve this conflict?

**Comment by eamanu** [comment]
@gabsprates  The problem (I think) is that you have your README.md very different to the @zeeshanu 's repo. Try to make a pull to update your repo and then do the PR

**Comment by gabsprates** [comment]
thanks @zeeshanu and @eamanu!

**Pull request merged**